src/test/utils.ts

```diff
@@ -35,7 +35,7 @@ export class TestProvider {
   }
 }
 
-export async function createTinlake(usr: Account, testConfig: any) : Partial<ITinlake> {
+export function createTinlake(usr: Account, testConfig: any) : Partial<ITinlake> {
```

Comment:
Can we add type info to the config Paramus and require the contract addresses to be present?